src/Sfa.Roatp.Register.Infrastructure/Elasticsearch/ElasticsearchClientFactory.cs

```diff
@@ -38,7 +38,7 @@ public IElasticClient Create()
             settings.DefaultMappingFor<Provider>(m => m
                 .IndexName("roatpproviderdocument"));
 
-            if (_applicationSettings.EnableES5)
+            if (!Debugger.IsAttached)
```

Comment:
What if you run without the debugger attached?
Would it be possible to use the EnvironmentName setting here?
/c